notebook/tests/launchnotebook.py

```diff
@@ -53,6 +53,8 @@ def wait_until_alive(cls):
         for _ in range(int(MAX_WAITTIME/POLL_INTERVAL)):
             try:
                 cls.fetch_url(url)
+            except ModuleNotFoundError as error:
+                raise ModuleNotFoundError(error)
```

Comment:
No need to recreate the error instance, let's just pass it on.

Also, could you please add a comment here, something somewhat generic like the following?  This way folks can extend this `except` clause as needed.
```suggestion
            except ModuleNotFoundError as error:
                # Errors that should be immediately thrown back to caller
                raise error
```